.pre-commit-config.yaml

```diff
@@ -7,7 +7,7 @@ repos:
     hooks:
       # Run the linter.
       - id: ruff
-        args: [ --fix ]
+        args: [ --fix,  --ignore, C901]
```

Comment:
Why ignore all C901 error (too complex code).  We can ignore specific files if needed.

Reply:
done